src/utilities/Readme.md

```diff
@@ -6,7 +6,9 @@ Green indicates the element is visible in your current viewport.
 <Row>
   <Col xs={6} sm={3}>
     <Hidden xs><div style={{ color: 'gray' }}>Extra small</div></Hidden>
-    <Visible xs><div style={{ color: 'green' }}>Visible on extra small</div></Visible>
+    <Visible xs portrait><div style={{ color: 'green' }}>Visible on extra small portrait</div></Visible>
```

Comment:
There are actually no `portrait` and `landscape` props for Visible. Probably it's better to update this example to use the `xl` screen class as well.